config/metadata/configMeta.yaml

```diff
@@ -1047,11 +1052,44 @@ groups:
           This list is ignored when Type is "redis". The format is a list of
           strings of the form "scheme://host:port".
 
+  - name: GooglePeerManagement
+    title: "Google Peer Management"
+    description: >
+      controls how the Refinery cluster communicates between peers when using
+      Google Pub/Sub. Only applies when `PeerManagement.Type` is "google".
+
+    fields:
+      - name: Topic
+        v1group: PeerManagement
+        v1name: GoogleTopic
+        type: string
+        valuetype: nonemptystring
+        example: ""
+        reload: false
+        envvar: REFINERY_GOOGLE_TOPIC
```

Comment:
If you want these envvars to work, they also have to be specified in cmdenv.go so they work on the command line too. IIRC, we don't read undecorated envvars separately. 

As of this month, we do allow string configuration values to look like `${ENVVAR}` which does substitute as expected.

Reply:
Thanks! The process of adding new config flags is a wee bit under-specified I fear so I did a fair bit of cargo cult copying and also doc regenerating.